pkg/deploy/api/helpers.go

```diff
@@ -17,3 +18,14 @@ func DeploymentToPodLogOptions(opts *DeploymentLogOptions) *kapi.PodLogOptions {
 		LimitBytes:   opts.LimitBytes,
 	}
 }
+
+// ScaleFromConfig builds a scale resource out of a deployment config.
+func ScaleFromConfig(dc *DeploymentConfig) *extensions.Scale {
+	return &extensions.Scale{
+		ObjectMeta: kapi.ObjectMeta{
+			Name:              dc.Name,
+			Namespace:         dc.Namespace,
+			CreationTimestamp: dc.CreationTimestamp,
```

Comment:
This timestamp looks unusual.  Why would we be forcing a `CreationTimestamp` on an object?  Doesn't that happen server-side in `FillObjectMetaSystemFields`?